.drone.yml

```diff
@@ -1,412 +1,2189 @@
+  pull: always
+  image: owncloud/ubuntu:16.04
+  detach: true
+  commands:
+  - tail -f /var/www/owncloud/data/owncloud.log
+
+- name: fix-permissions
+  pull: always
+  image: owncloudci/php:${PHP_VERSION=7.1}
+  commands:
+  - cd /var/www/owncloud/
+  - chown www-data * -R
+
+- name: webui-acceptance-tests
+  pull: always
+  image: owncloudci/php:${PHP_VERSION=7.1}
+  commands:
+  - cd /var/www/owncloud/phoenix
+  - curl http://phoenix:8300/oidc-callback.html
+  - yarn run acceptance-tests-drone
+  environment:
+    SCREEN_RESOLUTION: 375x812
+    TEST_TAGS: "@smokeTest and not @skipOnIphoneResolution and not @skip"
+
+- name: upload-screenshots
+  pull: if-not-exists
+  image: plugins/s3
+  settings:
+    acl: public-read
+    bucket: phoenix
+    endpoint: https://minio.owncloud.com/
+    path_style: true
+    source: /var/www/owncloud/phoenix/tests/reports/screenshots/**/*
+    strip_prefix: /var/www/owncloud/phoenix/tests/reports/screenshots
+    target: /screenshots/${DRONE_BUILD_NUMBER}
+  environment:
+    AWS_ACCESS_KEY_ID:
+      from_secret: aws_access_key_id
+    AWS_SECRET_ACCESS_KEY:
+      from_secret: aws_secret_access_key
+  when:
+    status:
+    - failure
+
+- name: build-github-comment
+  pull: always
+  image: owncloud/ubuntu:16.04
+  commands:
+  - cd /var/www/owncloud/phoenix/tests/reports/screenshots/
+  - "echo \"<details><summary>:boom: Acceptance tests <strong>${TEST_CONTEXT}</strong> failed. Please find the screenshots inside ...</summary>\\n\\n${DRONE_BUILD_LINK}/${DRONE_JOB_NUMBER}\\n\\n<p>\\n\\n\" >> comments.file"
+  - for f in *.png; do echo '!'"[$f](https://minio.owncloud.com/phoenix/screenshots/${DRONE_BUILD_NUMBER}/$f)" >> comments.file; done
+  - echo "\n</p></details>" >> comments.file
+  - more comments.file
+  when:
+    status:
+    - failure
+
+- name: github-comment
+  pull: if-not-exists
+  image: jmccann/drone-github-comment:1
+  settings:
+    message_file: /var/www/owncloud/phoenix/tests/reports/screenshots/comments.file
+  environment:
+    PLUGIN_API_KEY:
+      from_secret: plugin_api_key
+  when:
+    status:
+    - failure
+
+services:
+- name: phoenix
+  pull: always
+  image: owncloudci/php:${PHP_VERSION=7.1}
+  commands:
+  - mkdir dist
+  - make run SERVER_HOST=phoenix:8300
+
+- name: owncloud
+  pull: always
+  image: owncloudci/php:${PHP_VERSION=7.1}
+  command:
+  - /usr/local/bin/apachectl
+  - -e
+  - debug
+  - -D
+  - FOREGROUND
+  environment:
+    APACHE_WEBROOT: /var/www/owncloud/
+
+- name: selenium
+  pull: always
+  image: selenium/standalone-chrome-debug:latest
+
+- name: mysql
+  pull: if-not-exists
+  image: mysql:5.5
+  environment:
+    MYSQL_DATABASE: owncloud
+    MYSQL_PASSWORD: owncloud
+    MYSQL_ROOT_PASSWORD: owncloud
+    MYSQL_USER: owncloud
+
+trigger:
+  ref:
+  - refs/tags/**
+  - refs/pull/**
+  - refs/pull-requests/**
+  - refs/merge-requests/**
+  - refs/heads/master
+  - refs/heads/release*
+  - refs/heads/develop*
+
+---
+kind: pipeline
+name: matrix-14
```

Comment:
rename missing - maybe publish-npm-and-demo-system

should only be executed on push

Reply:
done